Makefile

```diff
@@ -130,9 +131,10 @@ tmp/ekam-bin: tmp/.deps
 	     cd ../.. && ln -s ../deps/ekam/bin/ekam-bootstrap tmp/ekam-bin)
 
 tmp/.ekam-run: tmp/ekam-bin src/sandstorm/* tmp/.deps
-	$(call color,building sandstorm with ekam)
+	@$(call color,building sandstorm with ekam)
 	@CC="$(CC)" CXX="$(CXX)" CFLAGS="$(CFLAGS)" CXXFLAGS="$(CXXFLAGS2)" \
-	    LIBS="$(LIBS)" NODEJS=$(NODEJS) tmp/ekam-bin -j$(PARALLEL)
+	    LIBS="$(LIBS)" NODEJS=$(NODEJS) tmp/ekam-bin -j$(PARALLEL) || \
+	$(call color,ekam failed. You might need to: make update-deps) && false
```

Comment:
Indent to make clear that this is a continuation of the previous line.

Maybe say "build failed" rather than "ekam failed" to avoid confusing people who don't know what ekam is?